Documentation/build-arm64-firmware.md

```diff
@@ -33,10 +33,9 @@ This document describes how to set up a build environment to build the latest fi
     $ git clone -b imx_4.14.62_1.0.0_beta https://source.codeaurora.org/external/imx/imx-atf
     $ git clone -b imx_4.14.62_1.0.0_beta https://source.codeaurora.org/external/imx/imx-mkimage
     ```
-    Optionally, clone the TPM reference implementation (`mu_platform_nxp` includes a precompiled TPM binary)
+    Optionally, clone the security TA repo (`mu_platform_nxp` includes precompiled TA binaries)
     ```bash
-    $ git clone --recursive https://github.com/Microsoft/ms-tpm-20-ref
-    $ pushd ms-tpm-20-ref; git checkout 65b65354c6cce3212d9c512ec3ae2e23fe37c94d; popd
+    $ git clone  https://github.com/Microsoft/MSRSec
```

Comment:
Should this be git clone --recursive?

git submodule status on MSRSec shows me the following:
D:\github\MSRSec>git submodule status
-6cb570e90317c7927a0c6da86a27777376a9a433 external/ms-tpm-20-ref
-3d753b0cefaa7e3d4b5d12d7805b20fabff1f385 external/ossl/openssl
-74ebf510a3d73e98767eac26082eabdc84e19d31 external/wolfssl

Reply:
OpenSSL has multiple massive submodules inside it which I wanted to avoid. The makefile in the MSRSec repo will pull down the required submodules at build time depending on which crypto package is selected.